website/docs/reference/resource-configs/athena-configs.md

```diff
@@ -0,0 +1,604 @@
+       'pi' as name,
+       'active' as status,
+       17.89 as cost,
+       1 as quantity,
+       100000000 as quantity_big,
+       current_date as my_date
+```
+
+`update_condition` example:
+
+```sql
+{{ config(
+        materialized='incremental',
+        incremental_strategy='merge',
+        unique_key=['id'],
+        update_condition='target.id > 1',
+        schema='sandbox'
+    )
+}}
+
+{% if is_incremental() %}
+
+select * from (
+    values
+    (1, 'v1-updated')
+    , (2, 'v2-updated')
+) as t (id, value)
+
+{% else %}
+
+select * from (
+    values
+    (-1, 'v-1')
+    , (0, 'v0')
+    , (1, 'v1')
+    , (2, 'v2')
+) as t (id, value)
+
+{% endif %}
+```
+
+Example of `insert_condition`:
+
+```sql
+{{ config(
+        materialized='incremental',
+        incremental_strategy='merge',
+        unique_key=['id'],
+        insert_condition='target.status != 0',
+        schema='sandbox'
+    )
+}}
+
+select * from (
+    values
+    (1, 0)
+    , (2, 1)
+) as t (id, status)
+
+```
+
+### High availablity table (HA)
+
+The current implementation of table materialization can lead to downtime, as the target table is dropped and re-created. To have less destructive behavior, it's possible to use the `ha` config on your `table` materialized models. It leverages the table versions feature of the glue catalog, creating a temp table and swapping the target table to the location of the temp table. This materialization is only available for `table_type=hive` and requires using unique locations. For Iceberg, high availability is the default.
+
+
+```sql
+{{ config(
+    materialized='table',
+    ha=true,
+    format='parquet',
+    table_type='hive',
+    partitioned_by=['status'],
+    s3_data_naming='table_unique'
+) }}
+
+select 'a'      as user_id,
+       'pi'     as user_name,
+       'active' as status
+union all
+select 'b'        as user_id,
+       'sh'       as user_name,
+       'disabled' as status
+```
+
+By default, the materialization keeps the last 4 table versions,but you can change it by setting `versions_to_keep`.
+
+#### HA known issues
+
+- When swapping from a table with partitions to a table without (and the other way around), there could be a little
+  downtime. If high performances is needed consider bucketing instead of partitions.
+- By default, Glue "duplicates" the versions internally, so the last two versions of a table point to the same location.
+- It's recommended to set `versions_to_keep` >= 4, as this will avoid having the older location removed.
+
+### Update glue data catalog
+
+Persist resource descriptions as column and relation comments to the glue data catalog, and meta as [glue table properties](https://docs.aws.amazon.com/glue/latest/dg/tables-described.html#table-properties) and [column parameters](https://docs.aws.amazon.com/glue/latest/webapi/API_Column.html). By default, documentation persistence is disabled, but it can be enabled for specific resources or groups of resources as needed.
```

Comment:
i'm having a hard time reading and grokking the first sentence. sugg rewording to improve clarity 

sugg updating second sentence to be in active voice

Reply:
Yeah, I'm struggling here too. @amychen1776 Any ideas on how to clean this thought up?

Reply:
Maybe something like this:

You can persist your column and model level descriptions to the Glue Data Catalog as [glue table properties](https://docs.aws.amazon.com/glue/latest/dg/tables-described.html#table-properties) and [column parameters](https://docs.aws.amazon.com/glue/latest/webapi/API_Column.html). To enable this, you will need to set the configuration to true as show below.